src/components/FormattedText/FormattedText.test.tsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import FormattedText from "./FormattedText";
+
+it("should render a text with a default font-size of 16px", () => {
+  const { getByText } = render(<FormattedText>My text</FormattedText>);
+
+  const mainText = getByText("My text");
+  expect(mainText).toHaveStyle({ fontSize: "16px" });
+});
+
+test.each([
```

Comment:
You get a type error here. Unfortunately the types on `jest.each` are not perfect, so we need to make it a bit more explicit.
```suggestion
import FormattedText, { Props } from "./FormattedText";

it("should render a text with a default font-size of 16px", () => {
  const { getByText } = render(<FormattedText>My text</FormattedText>);

  const mainText = getByText("My text");
  expect(mainText).toHaveStyle({ fontSize: "16px" });
});

test.each<[Props['size'], string]> ([
```

Reply:
```typescript
test.each<[Props['size'], string]> ([
```
this is just saying that each row of the tests will be a tuple of (size, string).

Reply:
Thanks Mo! Came across this error yesterday and thought it was something related to the types but wasn't sure of how to fix it.